### PR TITLE
Added TLS notes

### PR DESCRIPTION
**Pull request by dgarcia360**
Added some notes on how to add the VRF key and generate the TLS certificates.

### CODE REVIEW

File: docs/RUNNETWORKLIN.md [hunk 1]
```diff
@@ -151,7 +152,7 @@ The network mosaic ids are autogenerated based on the configuration provided in
 
 1\. Run the nemesis block generator.
 
-```ssh
+```sh
 ./catapult.tools.nemgen --nemesisProperties ../resources/mijin-test.properties
 ```
```
Comment (Jaguar0625) [inline docs/RUNNETWORKLIN.md:158]: line 161 ssh => sh

File: docs/RUNPEERLIN.md [hunk 2]
```diff
@@ -16,46 +15,39 @@ If you have not launched a network yet, move directly to ["Edit the node propert
...
 Learn more about each network property in [this guide](https://nemtech.github.io/guides/network/configuring-node-properties.html#properties).
 
-1\. Open ``resources/config-node.properties`` and search the ``[localnode]`` section.
+Open ``resources/config-node.properties`` and search the ``[localnode]`` section.
 Then, edit the properties with the peer node details.
 
 ``` ini
 [localnode]
-host = <YOUR_NODE_IP>
+host = 127.0.0.1
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:40]: why did you change this?

File: docs/RUNPEERLIN.md [hunk 3]
```diff
@@ -16,46 +15,39 @@ If you have not launched a network yet, move directly to ["Edit the node propert
...
-NOTE: You can generate new accounts with the command ./bin/catapult.tools.address -g 10 --network mijin-test.
-
 ## Enable harvesting
 
-This step enables harvesting, which allows the node to produce new blocks.
-At least a node of the network must have harvesting enabled in order to produce blocks.
+This step enables [harvesting](https://nemtech.github.io/concepts/harvesting.html), which allows the node to produce new blocks.
 
-NOTE: If you don't want to enable harvesting, move directly to [Add other peer nodes](#add-other-peer-nodes).
+NOTE: At least a node of the network must have harvesting enabled to produce new blocks. If you don't want to enable harvesting, move directly to [Add other peer nodes](#add-other-peer-nodes).
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:50]: a node => one node

File: docs/RUNPEERLIN.md [hunk 4]
```diff
@@ -66,62 +58,95 @@ extension.harvesting = true
...
 ```
 
-2\. Edit the file ``resources/config-harvesting.properties`` and replace ``harvestKey`` with the private key of an account.
-This account must have at least ``resources/config-network.properties#minHarvesterBalance`` harvest mosaic units.
+2\. Edit the file ``resources/config-harvesting.properties``.
 
 ```ini
 [harvesting]
-harvestKey = 0000000000000000000000000000000000000000000000000000000000000000
+harvesterSigningPrivateKey = ``<HARVESTER_PRIVATE_KEY>``
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:65]: SIGNING_PRIVATE_KEY ?

File: docs/RUNPEERLIN.md [hunk 5]
```diff
@@ -66,62 +58,95 @@ extension.harvesting = true
...
+harvesterVrfPrivateKey = ``<VRF_PIRVATE_KEY>`` 
 isAutoHarvestingEnabled = true
 ...
 ```
 
+* Replace ``<HARVESTER_PRIVATE_KEY>`` with the private key of an [elegible account](https://nemtech.github.io/concepts/harvesting.html#eligibility-criteria).
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:71]: eligible

File: docs/RUNPEERLIN.md [hunk 6]
```diff
@@ -66,62 +58,95 @@ extension.harvesting = true
...
 
-2\. Edit the file ``resources/config-harvesting.properties`` and replace ``harvestKey`` with the private key of an account.
-This account must have at least ``resources/config-network.properties#minHarvesterBalance`` harvest mosaic units.
+2\. Edit the file ``resources/config-harvesting.properties``.
 
 ```ini
 [harvesting]
-harvestKey = 0000000000000000000000000000000000000000000000000000000000000000
+harvesterSigningPrivateKey = ``<HARVESTER_PRIVATE_KEY>``
+harvesterVrfPrivateKey = ``<VRF_PIRVATE_KEY>``
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:66]: PRIVATE

File: docs/RUNPEERLIN.md [hunk 7]
```diff
@@ -66,62 +58,95 @@ extension.harvesting = true
...
-harvestKey = 0000000000000000000000000000000000000000000000000000000000000000
+harvesterSigningPrivateKey = ``<HARVESTER_PRIVATE_KEY>``
+harvesterVrfPrivateKey = ``<VRF_PIRVATE_KEY>`` 
 isAutoHarvestingEnabled = true
 ...
 ```
 
+* Replace ``<HARVESTER_PRIVATE_KEY>`` with the private key of an [elegible account](https://nemtech.github.io/concepts/harvesting.html#eligibility-criteria).
+
+* Replace ``<VRF_PIRVATE_KEY>`` with a private key linked to the harvester account to randomize the block production and leader selection. The link could be defined in the [nemesis block](RUNNETWORKLIN.md#append-the-vrf-keys-to-the-nemesis-block) or at a later point by announcing a **VRFKeyLinkTransaction** with the [CLI](https://github.com/nemtech/symbol-cli/blob/gh-pages/0.20.3.md#vrfkeylink) or [SDKs](https://github.com/nemtech/symbol-sdk-typescript-javascript).
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:73]: a private key => the private key

Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:73]: ~~and leader selection~~

File: docs/RUNPEERLIN.md [hunk 8]
```diff
@@ -66,62 +58,95 @@ extension.harvesting = true
...
+
+## Generate TLS certificates
+
+Catapult uses TLS 1.3 to provide secure connections and identity assurance between the nodes.
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:77]: between the => among all

File: docs/RUNPEERLIN.md [hunk 9]
```diff
@@ -66,62 +58,95 @@ extension.harvesting = true
...
+
+1. To generate and self sign the certificates, you can download and run the script [cert-generate.sh](https://github.com/tech-bureau/catapult-service-bootstrap/blob/master/common/ruby/script/cert-generate.sh) from the ``catapult-server/_build`` directory.
+
+```sh
+mkdir certificate
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:82]: certificates ?

File: docs/RUNPEERLIN.md [hunk 10]
```diff
@@ -66,62 +58,95 @@ extension.harvesting = true
...
+                "name": "peernode",
+                "roles": "Peer"
+            }
+        }
     ]
-   }
+}
+```
+
+To get the node public key, run the next command under the folder that contains the node's certificates:
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:126]: next => following
under => in

File: docs/RUNPEERLIN.md [hunk 11]
```diff
@@ -66,62 +58,95 @@ extension.harvesting = true
...
+
+```sh
+openssl pkey -pubin -in ca.pubkey.pem -noout -text | openssl dgst -sha256 -hex
 ```
 
 2\. If the network has more peer nodes, you can list them by adding new entries to the ``knownPeers`` array.
```
Comment (Jaguar0625) [inline docs/RUNPEERLIN.md:132]: you can list them by adding new entries to the => you can add them to ?

**Comment by dgarcia360** [comment]
@Jaguar0625 Thanks for the comments!

File: docs/RUNPEERLIN.md (hunk 4 above)
Comment (Jaguar0625) [reply docs/RUNPEERLIN.md]: i would use HARVESTER prefix either for both or none

**Comment by Jaguar0625** [comment]
consider using HARVESTER prefix for both or none of  { HARVESTER_SIGNING_PRIVATE_KEY, VRF_PRIVATE_KEY }
